src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -368,9 +368,8 @@ export class InstructorCourseEditPageComponent implements OnInit {
     this.courseService.updateCourse(this.courseToEdit.id, {
       courseName: newName,
       timeZone: newTimeZone,
-    }).subscribe((course: Course) => {
-      this.statusMessageService.showSuccessMessage(`Updated course [${course.courseId}] details: `
-          + `Name: ${course.courseName}, Time zone: ${course.timeZone}`);
+    }).subscribe(() => {
+      this.statusMessageService.showSuccessMessage('The course has been updated.');
```

Comment:
The current users are more used to seeing `The course has been edited.` and there's no good enough reason to change the phrasing.

What do you think? :)

Reply:
I agree. I see no reason to unnecessarily temper user experience